apps/foundation/pages/blog/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { Image, SimpleGrid, Stack, Text } from '@chakra-ui/react'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { dehydrate, QueryClient } from 'react-query'
+
+import { AnimatedBox, BlogCard, Container, Hero } from '@wsvvrijheid/ui'
+import { getBlogs, useGetBlogs } from '@wsvvrijheid/utils'
+import { Layout } from 'apps/foundation/components'
```

Comment:
I know VSCode adds the path automatically, it seems to be working with no issues. We can consider adding aliases for internal path imports but until then I would prefer to use relative imports

```suggestion
import { Layout } from '../../../components'
```